node/socketstream/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "todomvc-socketstream",
+  "description": "SocketStream TodoMVC app",
+  "author": {
+    "name": "Sindre Sorhus",
+    "email": "sindresorhus@gmail.com",
+    "url": "http://sindresorhus.com"
+  },
+  "private": true,
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "scripts": {
+    "start": "node app.js"
```

Comment:
I know but when i ran npm install bower install didn't work, causing missing files. hence added it

Get Outlook for iOS<https://aka.ms/o0ukef>




On Fri, Jun 3, 2016 at 12:46 PM -0700, "Arijit Layek" <notifications@github.com<mailto:notifications@github.com>> wrote:


In node/socketstream/package.json<https://github.com/alayek/zeus/pull/63#discussion_r65765158>:

> @@ -0,0 +1,21 @@
> +{
> +  "name": "todomvc-socketstream",
> +  "description": "SocketStream TodoMVC app",
> +  "author": {
> +    "name": "Sindre Sorhus",
> +    "email": "sindresorhus@gmail.com",
> +    "url": "http://sindresorhus.com"
> +  },
> +  "private": true,
> +  "engines": {
> +    "node": ">=0.10.0"
> +  },
> +  "scripts": {
> +    "start": "node app.js"


Add postinstall here as bower install. Then no need to run bower install separately!!

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/alayek/zeus/pull/63/files/8de62d60e1d28cb61613f9acd86ca9bacabed784#r65765158>, or mute the thread<https://github.com/notifications/unsubscribe/AE4rBg2HMej-DXDF76OcAoPqP6yYMhdLks5qIISmgaJpZM4IttOn>.